releasenotes/notes/remove-deprecated-quantumcircuit-a8c69a8e0b480e17.yaml

```
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    Removed remove features deprecated in 0.45 for `qiskit/circuit/quantumcircuit`:
```

Sorry I didn't catch this earlier:
```suggestion
    Removed features deprecated in the 0.45 release for :class:`.QuantumCircuit`:
```